randovania/gui/main_window.py

```diff
@@ -764,10 +764,21 @@ async def _on_menu_action_verify_installation(self):
         if bad_files or extra_files:
             errors = []
             if bad_files:
-                errors.append(f"- {len(bad_files)} files are missing or are incorrect")
+                errors.append(f"- {len(bad_files)} files are incorrect")
+
+            if missing_files:
+                errors.append(f"- {len(missing_files)} files are missing")
+
             if extra_files:
                 errors.append(f"- {len(extra_files)} files are unexpected")
```

Comment:
maybe its just me, but can you change these to be `* {len(extra_files=}` instead of using a hyphen (or if very fancy, use some unicode bullet character)? the first time i read `- 15 files are missing or are incorrect` i read it as `minus 15 files are missing or incorrect`.

Reply:
![image](https://github.com/randovania/randovania/assets/884928/c227189f-2c8d-4555-98e9-089fd10acab0)
it doesn't look very confusing for me 🤔 
but I can use `*` or something else

Reply:
Its more prominent when its only one list entry
![grafik](https://github.com/randovania/randovania/assets/38186597/f183dc07-f47b-4e25-97b7-5b98008fb283)

Reply:
I made this: https://github.com/randovania/randovania/issues/5520